client/controllers/incidentReports/newIncidentFromSelection.coffee

```diff
@@ -59,11 +60,14 @@ Template.newIncidentFromSelection.helpers
     _getAnnotationData(instance.selection).text
```

Comment:
We may not need to create a function for this since its only called once after your update?